drv/gimlet-hf-api/src/lib.rs

```diff
@@ -45,4 +45,16 @@ pub enum HfDevSelect {
     Flash1 = 1,
 }
 
+/// Size in bytes of a single page of data (i.e., the max length of slice we
+/// accept for `page_program()` and `read()`).
+// Note: There is no static check that this matches what's in our idl file in
+// terms of _client_ generation, but the server can use this constant in its
+// trait impl, which will produce a compile-time error if it doesn't match the
+// length in the idl file.
+pub const PAGE_SIZE_BYTES: usize = 256;
+
+/// Size in bytes of a single sector of data (i.e., the size of the data erased
+/// by a call to `sector_erase()`).
+pub const SECTOR_SIZE_BYTES: usize = 65_536;
```

Comment:
This came up [in another PR](https://github.com/oxidecomputer/hubris/pull/754#discussion_r965240718) - maybe this should actually be in `drv-stm32-qspi` and re-exported here instead?

Reply:
I mostly agree, with a small asterisk that it may not be consistent for all QSPI flashes (e.g. there are variants of the Sidecar flash which use 256K sectors).  Putting it in `drv-stm32-qspi` is not _worse_, though, and would let us use it in multiple places.